lib/stripe/api_resource.rb

```diff
@@ -48,6 +48,30 @@ def self.save_nested_resource(name)
       end
     end
 
+    # Adds a custom method to a resource class. This is used to add support for
+    # non-CRUDL API requests, e.g. capturing charges. custom_method takes the
+    # following parameters:
+    # - name: the name of the custom method to create (as a symbol)
+    # - http_verb: the HTTP verb for the API request (:get, :post, or :delete)
+    # - http_path: the path to append to the resource's URL. If not provided,
+    #              the name is used as the path
+    #
+    # For example, this call:
+    #     custom_method :capture, http_verb: post
+    # adds a `capture` class method to the resource class that, when called,
+    # will send a POST request to `/v1/<object_name>/capture`.
```

Comment:
technically it's `/v1/<object_name>s/capture` with the extra `.`. Because I like to be technically correct. :p

Reply:
![48c](https://user-images.githubusercontent.com/23086931/55270549-497ee580-525d-11e9-833d-bb13e244b07a.jpg)